Remove mis-exposed abort API on ProcessGroup

mrshenli opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32292 Remove mis-exposed abort API on ProcessGroup**

Differential Revision: [D19430252](https://our.internmc.facebook.com/intern/diff/D19430252)

mrshenli committed:
Remove mis-exposed abort API on ProcessGroup

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 43bdeac1:


**None of the build failures appear to be your fault**.

* **9/9** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=43bdeac1e291ab22fa10a8bc504935b64177a0bd).

### :snowflake: 9 failures recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270904) pytorch_linux_xenial_cuda9_cudnn7_py3_build (1/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87089771) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87089771)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270903) pytorch_linux_xenial_py3_6_gcc5_4_build (2/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87089983) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87089983)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270912) pytorch_linux_xenial_py3_clang5_asan_build (3/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090041) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090041)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270900) pytorch_xla_linux_xenial_py3_6_clang7_build (4/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090042) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090042)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-clang7:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-clang7:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-clang7:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270898) pytorch_linux_xenial_py3_clang5_android_ndk_r19c_x86_32_build (5/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090141) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090141)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270905) pytorch_linux_xenial_py3_clang5_android_ndk_r19c_mobile_code_analysis (6/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090142) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090142)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270894) pytorch_libtorch_linux_xenial_cuda9_cudnn7_py3_build (7/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090143) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090143)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270890) pytorch_linux_xenial_py3_clang5_mobile_build (8/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090144) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090144)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4270907) pytorch_linux_xenial_py2_7_9_build (9/9)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87090428) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87090428)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py2.7.9:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py2.7.9:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py2.7.9:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

mrshenli committed:
Update on "Remove mis-exposed abort API on ProcessGroup"


Differential Revision: [D19430252](https://our.internmc.facebook.com/intern/diff/D19430252)

[ghstack-poisoned]

mrshenli commented:
lint failures are on lines I didn't touch

pietern reviewed (approved):
Very odd -- thanks for catching this @mrshenli!!

mrshenli committed:
Update on "Remove mis-exposed abort API on ProcessGroup"


Differential Revision: [D19430252](https://our.internmc.facebook.com/intern/diff/D19430252)

[ghstack-poisoned]
